wifiphisher/common/phishinghttp.py

```diff
@@ -20,6 +23,48 @@ def get(self):
         self.redirect("http://10.0.0.1:8080/")
 
 
+class ValidateHandler(tornado.web.RequestHandler):
+    """
+    Validate the POST requests from client by the uimethods
+    """
+
+    def initialize(self, em):
+        """
+        :param self: A tornado.web.RequestHandler object
+        :param em: An extension manager object
+        :type self: tornado.web.RequestHandler
+        :type em: ExtensionManager
+        :return: None
+        :rtype: None
+        """
+
+        self.em = em
+
+    def post(self):
+        """
+        :param self: A tornado.web.RequestHandler object
+        :type self: tornado.web.RequestHandler
+        :return: None
+        :rtype: None
+        ..note: overide the post method to do the verification
+        """
+
+        json_obj = json_decode(self.request.body)
+        response_to_send = {}
+        verifications = {func.__name__: func for func in
+                         self.em.get_ui_funcs()}
+        # loop all the required verification methods
+        for key in list(json_obj.keys()):
+            if key in verifications:
+                # fire the corresponding varification method
+                result = getattr(uimethods, key)(json_obj[key])
```

Comment:
@sophron 
That's a good idea! It makes our implementation more compact, thanks for the suggestion.